### PR TITLE
[#5383] Add table property commands to Gravitino CLI

### PR DESCRIPTION
**Pull request by justinmclean**
### What changes were proposed in this pull request?

Added table property commands to Gravitino CLI.

### Why are the changes needed?

The expand the CLI support.

Fix: #5383

### Does this PR introduce _any_ user-facing change?

No, but it adds more commands.

### How was this patch tested?

Tested locally with Hive, Postgres and MySQL databases. Note that database support varies.

### CODE REVIEW

File: clients/cli/src/main/java/org/apache/gravitino/cli/commands/ListTableProperties.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+/** List the properties of a table. */
+public class ListTableProperties extends ListProperties {
+
+  protected final String metalake;
+  protected final String catalog;
+  protected final String schema;
+  protected final String table;
+
+  /**
+   * List the properties of a catalog.
```
Comment (shaofengshi) [inline clients/cli/src/main/java/org/apache/gravitino/cli/commands/ListTableProperties.java:41]: also some minor typos in this PR, please take a look

File: clients/cli/src/main/java/org/apache/gravitino/cli/commands/ListTableProperties.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      String table) {
+    super(url, ignoreVersions);
+    this.metalake = metalake;
+    this.catalog = catalog;
+    this.schema = schema;
+    this.table = table;
+  }
+
+  /** List the properties of a table. */
+  public void handle() {
```
Comment (shaofengshi) [inline clients/cli/src/main/java/org/apache/gravitino/cli/commands/ListTableProperties.java:65]: pls add Override annotation

**Comment by justinmclean** [comment]
Note this is failing due to an unrelated change in catalogs/catalog-lakehouse-paimon

**Comment by justinmclean** [comment]
@shaofengshi ready again

**Review by shaofengshi — APPROVED** [review]
LGTM

**Pull request merged**